app/timetables/models.py

```diff
@@ -39,3 +39,28 @@ def clean(self):
 
     def __str__(self):
         return self.name
+
+class Timetable(ForceCapitalizeMixin, TimestampMixin):
+    """
+    A timetable is the central entity or model of the platform.
+
+    It represents/encapsulates the entire structure and
+    scheduling of meals, menu-items, dishes, courses, options etc,
+    served at a location, to a team or the entire organisation.
+    """
+
+    name = models.CharField(max_length=255, unique=True)
+    code = models.CharField(max_length=60, unique=True)
+    api_key = models.CharField(max_length=255, unique=True)
+    cycle_length = models.IntegerField()
+    current_cycle_day = models.IntegerField()
+
+    capitalized_field_names = ('name',)
```

Comment:
@andela-joyebanji just like I noted in my comment on the dish model, the `ForceCapitalizeMixin` is also not appropriate for this model. The names of timetables could be complex and possibly include abbreviations e.g a timetable named 'HR Department' will be force to 'Hr department'. Not cool. Let's not force capitalization on the name field. We can still make it unique though.